doozer/doozerlib/backend/konflux_client.py

```diff
@@ -359,6 +360,9 @@ def _modify_param(params: List, name: str, value: Union[str, bool, list[str]]):
         _modify_param(params, "image-expires-after", "6w")
         _modify_param(params, "build-platforms", list(build_platforms))
 
+        if image_metadata.config.get("konflux", {}).get("network-mode") == "hermetic":
```

Comment:
looks like the hermetic build condition is the only thing we need from the Metadata object. Consider passing this around instead of the whole Metadata.

Reply:
either way,
/lgtm

Reply:
Have a couple of follow up PRs that might make use of metadata, so hoping to keep it in for now.

Thanks for the review